firmware/application/apps/ui_recon.cpp

```diff
@@ -1018,7 +1053,9 @@ void ReconView::on_statistics_update(const ChannelStatistics& statistics) {
             if (status != 1) {
                 status = 1;
                 if (wait != 0) {
-                    audio::output::stop();
+                    recon_stop_recording();
```

Comment:
I see this pattern a few times in the new code, would a helper function make sense?

Reply:
A few time, and slightly different. I thought it would add more bytes than save to factorize it.